src/closures.rs

```diff
@@ -1,3 +1,4 @@
+use rustc_ast::ast::StmtKind;
```

Comment:
I didn't realize that `ast` was already brought into scope below. You might consider just using `ast::StmtKind` instead.